doc/api/test.md

```diff
@@ -1483,6 +1512,11 @@ changes:
   does not have a name.
 * `options` {Object} Configuration options for the test. The following
   properties are supported:
+  * `bail` {boolean}
```

Comment:
I don't think this should be an option to `test()`. Should it be part of `run()` instead?

Reply:
Yes, my bad, the `bail` option it's part of `run`. I'll update the doc ASAP !

Reply:
fixed